templates/primary-secondary/primary-secondary.js

```diff
@@ -43,6 +45,8 @@ class TemplatePrimarySecondary extends LitElement {
 			}
 			.d2l-template-primary-secondary-footer {
 				grid-area: footer;
+				box-shadow: 0 -2px 4px rgba(86, 90, 92, .2);
+				height: 78px;

```

Comment:
I know the design says the height of this is `78px`, but I'm not sure if it actually should be. Our buttons are actually not a consistent height -- they grow/shrink based on the user's font size preference. So they're `42px` by default, but can be taller or shorter. Even with the largest font size, the buttons could probably still fit in `78px` (just with less top/bottom space), but it might be worth checking with design what the actual intent is?

Reply:
I checked with Daniel and it doesn't need to be a consistent 78px.